code/obj/item/device/radio.dm

```diff
@@ -1081,6 +1081,19 @@ obj/item/device/radio/signaler/attackby(obj/item/W as obj, mob/user as mob)
 	density = 0
 	desc = "A Loudspeaker."
 
+	New()
+		//..()
```

Comment:
why no `..()`?

Reply:
The parent `loudspeaker` currently has a deliberately empty `New()`.
In the case that the loudspeaker has a "populated" new in the future, it seems weird to me that the speakers should inherit it i.m.o.

Reply:
weird, and that doesn't call parents either.